drivers/mpu9250/MPU9250.cpp

```diff
@@ -250,8 +312,26 @@ int MPU9250::mpu9250_deinit()
 	return 0;
 }
 
+int MPU9250::pinThread(int cpu)
```

Comment:
I don't believe it's a good idea to have a thread inside the driver. The framework should take care of the scheduling. We had some discussions yesterday on the devcall about this as well. I think it needs a bigger discussion.